invenio_rdm_records/config.py

```diff
@@ -498,3 +498,10 @@ def make_doi(prefix, record):
 
 RDM_RESOURCE_ACCESS_TOKEN_REQUEST_ARG = "resource_access_token"
 """URL argument to provide resource access token."""
+
+RDM_RECORDS_ACCESS_REQUEST_TOKEN_LENGTH = None
```

Comment:
Judging from the documentation of `secrets.token_urlsafe`, it should be okay to have the standard lib determine the size for us:
```
Help on function token_urlsafe in module secrets:

token_urlsafe(nbytes=None)
    Return a random URL-safe text string, in Base64 encoding.
    
    The string has *nbytes* random bytes.  If *nbytes* is ``None``
    or not supplied, a reasonable default is used.
    
    >>> token_urlsafe(16)  #doctest:+SKIP
    'Drmhze6EPcv0fN_81Bj-nA'
```

The idea was to give admins a simple switch to increase the token length if they don't trust the defaults, given how fast processing power can grow.
However, since there's gonna be a follow-up issue for guest comments on the access requests anyway (which may include encoding the external email address into the token), we can always re-add such a functionality later if it's deemed relevant.